packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -256,7 +256,14 @@ class AnalyzeCommand extends FlutterCommand {
       mainFile.path
     ];
 
-    printStatus(cmd.join(' '));
+    if (argResults['congratulate']) {
```

Comment:
My workflow wants to have only errors if there's errors, but if there's no errors I need something to display so that I know the process didn't crash.